locale/ar/docs/guides/debugging-getting-started.md

```diff
@@ -171,8 +171,8 @@ layout: docs.hbs
 نوصي دائما بعدم تشغيل مصحح الأخطاء على عنوان انترنت عام. إذا أردت تمكين تصحيح الأخطاء عن بعد، فننصح بإستعمال قنوات الـ ssh بدلا من ذلك.
 المثال الآتي لأغراض توضيحية فقط. يجب عليك فهم المخاطر الأمنية المحتملة عند السماح بالوصول عن بعد لخدمة ذات امتيازات قبل أن تمضي قدما.
 
-فلنفترض أنك قمت بتشغيل الـ Node في حاسوب بعيد، remote.example.com على سبيل المثال، و تريد ان تتمكن من تصحيح الأخطاء فيها.
-في ذلك الحاسوب البعيد، عليك بدء عملية node مع جعل المدقق يستمع على المضيف المحلي فقط (وهو الافتراضي).
+فلنفترض أنك قمت بتشغيل الـ Node.js في حاسوب بعيد، remote.example.com على سبيل المثال، و تريد ان تتمكن من تصحيح الأخطاء فيها.
+في ذلك الحاسوب البعيد، عليك بدء عملية node.js مع جعل المدقق يستمع على المضيف المحلي فقط (وهو الافتراضي).
```

Comment:
Should be capitialized unless the original text was talking about the executable binary
```suggestion
في ذلك الحاسوب البعيد، عليك بدء عملية Node.js مع جعل المدقق يستمع على المضيف المحلي فقط (وهو الافتراضي).
```

Reply:
https://github.com/nodejs/nodejs.org/pull/2914#discussion_r371007977